files/ldap-stg/docker-compose.yml

```diff
@@ -1,37 +1,45 @@
 ---
-version: '3.5'
+version: '3.8'
+
+volumes:
+  openldap:
+    driver: local
 
 services:
   openldap:
     restart: "always"
-    image: osixia/openldap:1.5.0
-    command: "--copy-service"
+    image: bitnami/openldap:2.5.18
+    hostname: ldap-stg.openmrs.org
     ports:
       - 636:636
     environment:
-      LDAP_ORGANISATION: OpenMRS
-      LDAP_DOMAIN: openmrs.org
-      LDAP_ADMIN_PASSWORD: ${LDAP_ADMIN_PASSWORD:-admin}
-      LDAP_CONFIG_PASSWORD: ${LDAP_CONFIG_PASSWORD:-config}
-      LDAP_TLS: ${LDAP_TLS:-true}
-      LDAP_TLS_CRT_FILENAME: cert.pem
-      LDAP_TLS_KEY_FILENAME: privkey.pem
-      LDAP_TLS_CA_CRT_FILENAME: fullchain.pem
-      LDAP_TLS_VERIFY_CLIENT: never
-      LDAP_LOG_LEVEL: 0
+      - LDAP_ROOT=dc=openmrs,dc=org
+      - LDAP_ADMIN_DN=cn=admin,dc=openmrs,dc=org
+      - LDAP_TLS_VERIFY_CLIENT=never
+      - LDAP_ADMIN_USERNAME=admin
+      - LDAP_ADMIN_PASSWORD=${LDAP_ADMIN_PASSWORD:-admin}
+      - LDAP_CONFIG_ADMIN_ENABLED=yes
+      - LDAP_CONFIG_ADMIN_USERNAME=admin
+      - LDAP_CONFIG_ADMIN_PASSWORD=${LDAP_CONFIG_PASSWORD:-config}
+      - LDAP_LDAPS_PORT_NUMBER=636
+      - LDAP_PORT_NUMBER=389
+      - LDAP_ENABLE_TLS=${LDAP_ENABLE_TLS:-yes}
```

Comment:
I added this environment variable as it allows me to run this locally without TLS.